import.ts

```diff
@@ -7,6 +7,6 @@ import '@/modules/subscriptions/subscriptions'
 import '@/modules/templates/templates'
```

Comment:
This file shall not be commited to git.
Is generated on the `pnpm load` 
The pipeline shall executed it during the building process :D